test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,6 +36,18 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void isAnyNull() {
+        assertTrue(Utils.isAnyNull(1, 2, 3, null, 13));
+        assertTrue(Utils.isAnyNull(1, 2, null, null, 13));
+        assertTrue(Utils.isAnyNull(null, 1, 2, 3, 5));
+        assertTrue(Utils.isAnyNull(1, 2, 3, null));
+        assertTrue(Utils.isAnyNull((Object) null));
+        assertFalse(Utils.isAnyNull());
+        assertFalse(Utils.isAnyNull(new Object[]{}));
+        assertFalse(Utils.isAnyNull(1, 2, 3, 4, 5));
```

Comment:
Perhaps you can use comments to separate different test cases for better readability? For example,
// Non-empty list with nulls
assertTrue(Utils.isAnyNull(1,2 ,3, null, 13));
...

// Empty list
assertFalse(Utils.isAnyNull());
...